docs/src/architecture/08_concepts/immutable_ledger/cddl/block.cddl

```diff
@@ -0,0 +1,32 @@
+block = [
+    block_header,
+    block_data: ~encoded-cbor, ; deterministically encoded CBOR
```

Comment:
From our discussion on Friday, should this be bytes string?

Reply:
But it is a bytes string, `encoded-cbor` is a type alias of `#6.24(bstr)`.
Here it is how it's defined in the cddl spec  `Appendix D.  Standard Prelude` https://datatracker.ietf.org/doc/rfc8610/
```cddl
 encoded-cbor = #6.24(bstr)
```

Reply:
Okey, understood 🙌🏼